lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,184 @@
+$nx-sidebar-nav-background-color: $nx-blue-1000;
+$nx-sidebar-nav-text-color: $nx-white;
+$nx-sidebar-nav-help-color: $nx-grey-50;
+$nx-sidebar-nav-hover-color: $nx-indigo-600;
+$nx-sidebar-nav-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+
+.nx-sidebar-nav {
+  @include container-vertical;
+
+  box-sizing: border-box;
+  background-color: $nx-sidebar-nav-background-color;
+  color: $nx-sidebar-nav-text-color;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  min-width: 250px;
+  padding: 0;
+  padding-top: $nx-spacing-l;
+  position: relative;
+  // transition: all 0.5s ease;
+}
+
+.nx-sidebar-nav--product-info {
+  margin: $nx-spacing-sm $nx-spacing-l 0;
+  height: $nx-sidebar-logo-height;
+}
+
+.nx-sidebar-nav--product-name {
+  display: inline-block;
+  font-size: 16px;
+  line-height: $nx-sidebar-logo-height;
+  vertical-align: top;
+}
+
+.nx-sidebar-nav--logo {
+  height: $nx-sidebar-logo-height;
+  margin-right: $nx-spacing-xs;
+  width: 30px;
+}
+
+.nx-sidebar-nav--icon {
+  color: $nx-sidebar-nav-text-color;
+  margin: 0;
+}
+
+.nx-footer-help-icon {
+  color: $nx-sidebar-nav-help-color;
+}
+
+.nx-sidebar-nav--footer {
+  align-self: center;
+  bottom: $nx-spacing-xxl;
+  font-size: $nx-font-size-xs;
+  position: absolute;
+  text-align: center;
+}
+
+.nx-sidebar-nav--items {
+  list-style-type: none;
+  margin: 36px 0;
+  padding: 0;
+  width: 100%;
+}
+
+.nx-sidebar-nav--item {
+  height: 32px;
+
+  &:hover {
+    background-color: $nx-sidebar-nav-hover-color;
+  }
+
+  &.selected {
+    background-color: $nx-sidebar-nav-selected-color;
+
+    .nx-sidebar-nav--link {
+      @include semi-bold;
+    }
+  }
+}
+
+.nx-sidebar-nav--separator {
+  background-color: $nx-sidebar-separator-color;
+  height: 1px;
+  width: 100%;
+}
+
+.nx-sidebar-nav--release {
+  height: 36px;
+  margin-bottom: $nx-spacing-sm;
+  margin-top: $nx-spacing-xxl;
+}
+
+.nx-sidebar-nav--link {
+  @include regular;
+
+  color: inherit;
+  display: inline-block;
+  font-size: 16px;
+  height: 32px;
+  line-height: 32px;
+  text-decoration: none;
+
+  .nx-icon {
+    width: 21px;
+  }
+}
+
+.nx-sidebar-nav--help-link {
+  color: inherit;
+  font-size: 14px;
+  text-decoration: none;
+
+  .nx-icon {
+    margin-left: 0;
+  }
+}
+
+.nx-sidebar-nav--help-text {
+  font-size: 14px;
+}
+
+.nx-sidebar-nav--items {
+  margin-bottom: $nx-spacing-xxl;
+}
+
+.nx-sidebar-nav__open {
+  .nx-sidebar-nav--toggle {
+    height: 16px;
+    position: absolute;
+    right: $nx-spacing-l;
+    width: 16px;
+  }
+
+  .nx-sidebar-nav--item {
+    padding-left: $nx-spacing-l;
+  }
+
+  .nx-sidebar-nav--link-name {
+    margin-left: $nx-spacing-sm;
+  }
+}
+
+.nx-sidebar-nav__closed {
+  min-width: 69px;
+
+  .nx-sidebar-nav--toggle {
+    align-self: center;
+  }
+
+  .nx-sidebar-nav--logo {
+    display: none;
+  }
+
+  .nx-sidebar-nav--items {
+    text-align: center;
+  }
+
+  .nx-sidebar-nav--release {
+    width: 44px;
```

Comment:
magic number.  It appears this is here to force the release number to wrap before it gets too close to the edge.  Id' say that should be the responsibility of padding on the sidebar itself.

Reply:
> Id' say that should be the responsibility of padding on the sidebar itself.

Or maybe padding on the sidebar footer, if that would work better